src/Maestro/Microsoft.DotNet.Maestro.Tasks/Microsoft.DotNet.Maestro.Tasks.csproj

```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="Microsoft.Build" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" />
     <PackageReference Include="Microsoft.DotNet.VersionTools" />
+    <PackageReference Include="System.Security.Cryptography.Xml" />
```

Comment:
Please add a comment explaining we don't need this package but only pin it as it's a transitive dependency which is alerting in CG

Reply:
Added. The pipeline run was also didn't have any CG issues